### PR TITLE
make support for rails 3

### PR DESCRIPTION
**Pull request by trioglobal**
a little change that will make the gem work with rails 3

### CODE REVIEW

**Comment by rich311** [comment]
Can this be merged in?  I'd like to use this gem with rails 3 as well.

**Comment by senny** [comment]
@trioglobal changing the dependencies like this will break newer versions of Rails. if there are minimal code changes required to get `3.2` working we can discuss to use a `>=` version specifier. Note though that the library will not officially support Rails 3.2. So it's possible that support breaks again in future releases.

File: lib/pdfjs_viewer-rails.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ class Engine < ::Rails::Engine
       isolate_namespace PdfjsViewer
 
       initializer 'pdfjs_viewer-rails.load_static_assets' do |app|
-        app.middleware.unshift ::ActionDispatch::Static, "#{root}/public"
+        app.middleware.insert_before(::ActionDispatch::Static, ::ActionDispatch::Static, "#{root}/public")
```
Comment (senny) [inline lib/pdfjs_viewer-rails.rb]: Is there no other way to insert something at the beginning? What if `ActionDispatch::Static` is not part of the production middleware stack?

Comment (trioglobal) [inline lib/pdfjs_viewer-rails.rb]: we are moving away from rails 3, so if no one else needs it we can just delete the pull reg.

**Comment by Elektron1c97** [comment]
I'm going to close this, as @trioglobal said that they're moving away from rails 3.

Anyone who needs this engine in rails 3 can create a new pull request and use this as "inspiration".

**Pull request closed without merging**